### PR TITLE
4-2-RN

### PR DESCRIPTION
**Pull request by jenjoe22**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/enterprise-edition/rn/4-2-rn.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
```
Comment (Aishbn) [inline docs/en/enterprise-edition/rn/4-2-rn.yaml:1]: I think you should remove this file before merging.

**Review by Aishbn — COMMENTED** [review]
@jenjoe22 All the updates LGTM. However, please remove 4-2-rn.yml before merging the branch.

File: docs/en/enterprise-edition/rn/4-2-rn.yaml (hunk 1 above)
Comment (jenjoe22) [reply docs/en/enterprise-edition/rn/4-2-rn.yaml:1]: Yes I'll go ahead and remove the 4-2-rn.yml file.

File: docs/en/enterprise-edition/rn/prisma-cloud-release-info/features-introduced-in-2024/features-introduced-in-april-2024.adoc [hunk 2]
```diff
@@ -6,10 +6,10 @@ Learn what's new on Prisma® Cloud in April 2024.
 * <<api-ingestions>>
 * <<new-policies>>
 * <<policy-updates>>
-//* <<new-compliance-benchmarks-and-updates>>
+* <<new-compliance-benchmarks-and-updates>>
 * <<rest-api-updates>>
-//* <<changes-in-existing-behavior>>
-//* <<deprecation-notices>>
+* <<changes-in-existing-behavior>>
+* <<deprecation-notices>>
```
Comment (Aishbn) [inline docs/en/enterprise-edition/rn/prisma-cloud-release-info/features-introduced-in-2024/features-introduced-in-april-2024.adoc:12]: Please comment out `<<deprecation-notices>>` menu

**Review by Aishbn — APPROVED** [review]
@jenjoe22 I left a comment. Please fix it. Rest of the content LGTM.

**Pull request merged**